lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -206,7 +206,16 @@ for node in $allocated_nodes; do
 done
 
 # Make sure all openstack services are down on all nodes before we stop rabbitmq
-sleep 10
+services_stopped=""
+while [ -z $services_stopped ]; do
+  sleep 1
+  services_stopped="yes"
+  for node in $allocated_nodes; do
+    if ssh "$node" 'for i in /etc/init.d/openstack-* ; do if test -e $i && $i status >/dev/null ; then echo "$i still running at $node"; exit 0; fi; done; exit 1'; then
```

Comment:
`$node` won't get expanded inside the single quotes.

Also you can avoid the `test -e` by doing `shopt -s nullglob` outside the `for` loop.